src/core/lightning/index.ts

```diff
@@ -318,9 +323,13 @@ export const LightningMixin = (superclass) =>
             // if we failed to do it, the invoice would still be present in InvoiceUser
             // in case the invoice were to be paid another time independantly (unlikely outcome)
             try {
-              const { lnd } = getLndFromPubkey({ pubkey })
-
-              await cancelHodlInvoice({ lnd, id })
+              const lndService = LndService()
+              if (lndService instanceof Error) return lndService
+              const deleteResult = lndService.deleteUnpaidInvoice({
```

Comment:
I think Unpaid in `deleteUnpaidInvoice` may be a bit misleading. the invoice was paid if we are in this branch. just not through lnd but with intra ledger.

Reply:
Ok I can maybe use something like `cancelInvoice` instead?

Reply:
Done here: https://github.com/GaloyMoney/galoy/pull/581/commits/99d27842d1cd175ebee5580928804bde6e5bf0ca